cloudify_netconf/xml_rpc.py

```diff
@@ -501,16 +503,33 @@ def _run_calls(netconf, message_id, netconf_namespace, xmlns, calls,
             ctx.instance.runtime_properties[save_to + "_ns"] = xmlns
 
 
+def _get_template(template_location):
+    parse_result = urlparse(template_location)
+    if all([parse_result.scheme, parse_result.path]):
+       if parse_result.scheme == 'file':
+           with open(parse_result.path) as tmpl_f:
+               return tmpl_f.read()
+       else:
+           return requests.get(template_location).content
+    else:
+        return ctx.get_resource(template_location)
+
+
 @operation
 def run(**kwargs):
     """main entry point for all calls"""
 
     calls = kwargs.get('calls', [])
 
+    templates_urls = kwargs.get('templates', [])
     template = kwargs.get('template')
+
     templates = []
+    for tmpl_loc in templates_urls:
+        templates.append(_get_template(tmpl_loc))
```

Comment:
Do we support several templates per url? We support in embedded to blueprint templates several calls per template with `]]>]]>` mark.

Reply:
supported for "template" but not for "templates"